toolkit/docs/building/building.md

```diff
@@ -69,24 +68,33 @@
     - [Directory Customization](#directory-customization)
     - [Build Details](#build-details)
 
-## Quick Start
+## Overview
+
+The following documentation describes how to fully build CBL-Mariner from end-to-end as well as advanced techniques for performing toolchain, or package builds.  Full builds of CBL-Mariner _**is not**_ generally needed.  All CBL-Mariner packages are built signed and released to an RPM repository at [pacakages.microsoft.com](https://packages.microsoft.com/cbl-mariner/1.0/prod/)  
```

Comment:
>from end-to-end [](start = 69, length = 15)

Nit: I'm still not sure this is the correct way to write this.

Reply:
i think you are correct...the from is not needed.  Fixed.